include/deal.II/base/grid_tools.h

```diff
@@ -0,0 +1,429 @@
+// ---------------------------------------------------------------------
```

Comment:
This can't be right -- you've moved the whole file into the `base/` directory. It should stay in `distributed/`.